lang/cs/Org.Apache.REEF.IO/FileSystem/Local/LocalFileSystem.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Org.Apache.REEF.Tang.Annotations;
+
+namespace Org.Apache.REEF.IO.FileSystem.Local
+{
+    /// <summary>
+    /// An implementation of IFileSystem on the filesystem of the host.
+    /// </summary>
+    internal sealed class LocalFileSystem : IFileSystem
+    {
+        [Inject]
+        private LocalFileSystem()
+        {
+        }
+
+        public Stream Open(Uri fileUri)
+        {
+            return File.Open(fileUri.LocalPath, FileMode.Open);
+        }
+
+        public Stream Create(Uri fileUri)
```

Comment:
LocalFileSystem  is more like a wrapper of System File and Directory. What's the point to wrap them?